### ISSUE TITLE
missing source file in 6.0.0

### ISSUE THREAD

**Issue by antonio-rojas** [shlomif/fc-solve#47]
6.0.0 fails to configure with a missing source file error
```
CMake Error at CMakeLists.txt:587 (ADD_LIBRARY):
  Cannot find source file:

    fcs_is_ss_true_parent.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx
```

**Comment by shlomif** [comment]
@antonio-rojas : hi! Thanks for the report, but I'll need more details:

1. OS / distro / versions.

2. Which commands did you use  to build.

3. See: https://github.com/shlomif/writing-the-perfect-question

Note that the build seems fine on travis ( https://travis-ci.org/github/shlomif/fc-solve/jobs/710810869 ) and on appveyor ( https://ci.appveyor.com/project/shlomif/fc-solve/builds/34244393 ) and on the mageia buildsys ( http://pkgsubmit.mageia.org/ ) and locally.

**Comment by antonio-rojas** [comment]
> 1. OS / distro / versions.

Arch Linux

> 2. Which commands did you use  to build.

```
  cmake -B build -S freecell-solver-6.0.0 \
    -DCMAKE_INSTALL_PREFIX=/usr \
    -DFCS_WITH_TEST_SUITE=OFF \
    -DBUILD_STATIC_LIBRARY=OFF
  cmake --build build
```

**Comment by shlomif** [comment]
>   cmake -B build -S freecell-solver-6.0.0 \
>     -DCMAKE_INSTALL_PREFIX=/usr \
>     -DFCS_WITH_TEST_SUITE=OFF \
>     -DBUILD_STATIC_LIBRARY=OFF
>   cmake --build build

Thanks! I was able to reproduce it on fedora 32. Apparently I assumed that the build/binary directory is also the $PWD. "cd"ing to it before running cmake fixes this issue, e.g:

```bash
cmake -B "`pwd`" -S ../freecell-solver-6.0.0 \
    -DCMAKE_INSTALL_PREFIX=/usr \
    -DFCS_WITH_TEST_SUITE=OFF \
    -DBUILD_STATIC_LIBRARY=OFF
```

Thanks!

**Comment by antonio-rojas** [comment]
Fixed with a4a12a7abf039384b90d7f92f4259520af574594

**Comment by shlomif** [comment]
> Fixed with [a4a12a7](https://github.com/shlomif/fc-solve/commit/a4a12a7abf039384b90d7f92f4259520af574594)

Maybe, but it causes the travis-CI build to fail (see https://travis-ci.org/github/shlomif/fc-solve/jobs/710871424 ), was not released as a tarball yet and is not part of the main "master" branch. https://en.wikipedia.org/wiki/It_ain%27t_over_till_the_fat_lady_sings and https://biblehub.com/1_kings/20-11.htm . I'd like to try and correct the CI failure, but feel free to use the patch in the changeset for now.

**Comment by shlomif** [comment]
> > Fixed with [a4a12a7](https://github.com/shlomif/fc-solve/commit/a4a12a7abf039384b90d7f92f4259520af574594)
> 
> Maybe, but it causes the travis-CI build to fail (see https://travis-ci.org/github/shlomif/fc-solve/jobs/710871424 ), was not released as a tarball yet and is not part of the main "master" branch. https://en.wikipedia.org/wiki/It_ain%27t_over_till_the_fat_lady_sings and https://biblehub.com/1_kings/20-11.htm . I'd like to try and correct the CI failure, but feel free to use the patch in the changeset for now.

**Update**: travis-ci build is passing now: https://travis-ci.org/github/shlomif/fc-solve/builds/711055157 . AppVeyor build is still passing: https://ci.appveyor.com/project/shlomif/fc-solve . I'll try minting a new tarball release now.

**Comment by shlomif** [comment]
Should be fixed in release v6.0.1. Closing for now.
